### PR TITLE
test(e2e, playwright): tool-panel.tsx, added tests for buttons in 'Get Help'

### PR DESCRIPTION
**Pull request by weilirs**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

https://github.com/freeCodeCamp/freeCodeCamp/issues/51705

### CODE REVIEW

File: e2e/tool-panel.spec.ts [hunk 1]
```diff
@@ -50,4 +50,31 @@ test.describe('Tool Panel', () => {
 
     expect(helpLinks).toEqual(expectedHelpLinks);
   });
+
+  // "Watch a Video" is done by "video-modal.spec.ts"
+  test('check the buttons in "Get Help" dropdown except "Watch a Video"', async ({
```
[the remark this answers is not in the document]
Comment (weilirs) [reply e2e/tool-panel.spec.ts]: I made the changes.

File: e2e/tool-panel.spec.ts [hunk 2]
```diff
@@ -50,4 +50,31 @@ test.describe('Tool Panel', () => {
...
+    const browserContext = page.context();
+    const [newPage] = await Promise.all([
+      browserContext.waitForEvent('page'),
+      hintButton.click()
+    ]);
+    await newPage.waitForLoadState();
+    await expect(newPage).toHaveURL(
+      'https://forum.freecodecamp.org/t/freecodecamp-challenge-guide-increment-a-number-with-javascript/18201'
+    );
+    await newPage.close();
```
[the remark this answers is not in the document]
Comment (weilirs) [reply e2e/tool-panel.spec.ts]: I made the changes.

**Review by huyenltnguyen — APPROVED** [review]
Thank you for the contribution!

The Cypress failure is fixed via #52495. We will help update this PR once that one is merged :)

**Review by moT01 — APPROVED** [review]
LGTM @weilirs 🎉 Thanks for contributing to freeCodeCamp 🎉

**Pull request merged**